packages/app/cypress/e2e/runs.cy.ts

```diff
@@ -644,6 +644,7 @@ describe('App: Runs', { viewportWidth: 1200 }, () => {
         o.sinon.stub(ctx.config.electronApi, 'copyTextToClipboard')
       })
 
+      cy.get('[data-cy="terminal-prompt-input').should('have.value', 'npx cypress run --component --record --key 2aaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa')
```

Comment:
I assume it didn't have a value because we are querying for the record key?

Reply:
Yah, we're just clicking the button before the key loads in.